src/main.jl

```diff
@@ -76,6 +85,11 @@ function main(;
         mkdir(outdir)
         @info "Write output to $outdir"
     end
+
+    if Dates.Day(start_date) != Dates.Day(end_date) || Dates.Month(start_date) != Dates.Month(end_date)
```

Comment:
We could use the montday function here to check for both simultaneously 
```suggestion
    if monthday(start_date)  !=  monthday(end_date)
```